lib/debuglet.js

```diff
@@ -83,42 +84,44 @@ function Debuglet(config, logger) {
  */
 Debuglet.prototype.start = function(opt_callback) {
   var that = this;
-
-  uid.get(that.config_.workingDirectory, function(err, uid) {
-    if (err) {
-      that.logger_.error('Unable to compute unique id of the application', err);
-      if (opt_callback) {
-        opt_callback(err);
-      }
-      return;
-    }
-    that.logger_.info('Unique ID for this Application: ' + uid);
-
-    that.debugletApi_.init(uid, function(err) {
+  scanner.scan(that.config_.workingDirectory, function(err, hash, fileStats) {
```

Comment:
Maybe you can bring that logic into this file and refuse to enable the debulet unless we can find a package.json in `config.workingDirectory`.